apps/storybook/.storybook/preview.tsx

```diff
@@ -120,6 +120,10 @@ const components = {
 };
 
 const preview: Preview = {
+  argTypes: {
+    'data-color': { control: { type: 'text' } },
+    'data-size': { control: { type: 'text' } },
```

Comment:
```suggestion
    'data-size': { control: { type: 'text' }, options: ['sm', 'md', 'lg'], },
```
Could we add options here as well? Also make sure to override this for the heading and avatar stories, since those have more sizes

Reply:
you need a different type for options to work with argType control. but we'll have to test